### PR TITLE
Unit & Integration testing

### PR DESCRIPTION
**Pull request by KapilTR**
Added unit and integration test cases to review for accounts, contracts, transactions and setup.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "postinstall": "mkdir -p /tmp/tezster && cp -n ./config.json /tmp/tezster/config.json",
     "postinstall-win": "mkdir -p /tmp/tezster && copy -n ./config.json /tmp/tezster/config.json",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "mocha",
+    "test-with-coverage": "nyc mocha"
```
Comment (jvtezsure) [inline package.json]: rename it to 'test-coverage'

Comment (jvtezsure) [reply package.json]: you should cover files only meant for CLI (like tezster.js, tezster-manager.js ,module folder) and exclude files under lib or any other folder.

File: test/test.accounts.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
+const sinon = require("sinon"),
+      fs = require('fs'),
```
Comment (jvtezsure) [inline test/test.accounts.js:4]: remove unused variables.

File: test/test.contracts.js [hunk 3]
```diff
@@ -0,0 +1,312 @@
...
+            ophash = sendContractOriginationOperation.applied.results.contents[0].metadata.operation_result.originated_contracts;
+
+            stubAddNewContract = sandbox
+                                .stub(contract, 'addNewContract')
+                                .withArgs(CONTRACT_DEPLOY_LABEL, CONTRACT_ADDRESS, BOOTSTRAPPED_ACCOUNT, testconfig.provider);
+                                
+            await contract.deployContract([CONTRACT_DEPLOY_LABEL, './MichelsonCodeFile.tz', CONTRACT_INITIAL_STORAGE, BOOTSTRAPPED_ACCOUNT]);
+            sinon.assert.calledOnce(stubReadFileSync);
+            sinon.assert.calledOnce(stubConseil);
+            sinon.assert.calledOnce(stubAddNewContract);
```
Comment (jvtezsure) [inline test/test.contracts.js:112]: wherever a function returns a value, you can assert if returned value is same or not.

**Review by jvtezsure — CHANGES_REQUESTED** [review]

File: test/test.contracts.js (hunk 3 above)
Comment (KapilTR) [reply test/test.contracts.js:112]: yes, will do that, I forgot to include this. Other than this please review the code.

**Pull request closed without merging**